src/components/AppBar/components/HamburgerButton/hamburger.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="14" height="13" viewBox="0 0 14 9" xmlns="http://www.w3.org/2000/svg">
+<path d="M1 1H13" stroke="white" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M1 5H13" stroke="white" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M1 9H13" stroke="white" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
God, I love SVGs.

Reply:
Hm. I'm somewhat undecided on this. On the one hand upgrading projects is annoying work. At the same time a design system should create consistency and ideally allow consumers to get the latest look for free. I think I lean towards replacing the existing thing, if it's API compatible.